compiler/onecc-docker/onecc-docker_1.21.0_amd64/usr/bin/onecc-docker

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
```

Comment:
thoughts behind this. `Dockerfile` is infra like file. I don't think this should be placed inside `compiler`. `debian` files are aosl infra like files. these files are not directly related with compiler modules and thus looks more comfortable placed in `infra` folder.

Reply:
> (1) where is `onecc-docker/` placed? in the root?

This directory will be located in `./compiler/`.

> thoughts behind this: `Dockerfile` is infra like file. I don't think this should be placed inside `compiler`. `debian` files are also infra like files. these files are not directly related with compiler modules and thus looks more comfortable placed in `infra` folder.

What about `./infra/debian/onecc-docker`?

Reply:
> I'm curious how the build steps of compiler and debian package is going to be...
> and how are we goding to test in local?

Test step will use this code(#9731). 

And I'm considering how to work on `one-compiler` build and `onecc-docker` build.

Reply:
> Do you agree onecc-docker debian codes into ./infra/debina/onecc-docker...?

I wrote about this in https://github.com/Samsung/ONE/pull/9753#issuecomment-1253327416
- existing ones are `compiler` and `runtime`.
- level of introducing level seems different for `onecc-docker`
- so I wanted to give some level matching

Reply:
> I want to apply this code into ./infra/debian/compiler code. However, onecc-docker should be uploaded to launchpad for accessing in ONE-vscode. 

As I know this will make things difficult for existing codes to exlude for `onecc-docker`.

> So I want to work on the onecc-docker implementation separately from the existing one-compiler.

Me too :)

Reply:
In offline discussion with @seanshpark, we decided on the following Debian directory location.

`onecc-docker` debian directory will be located in `./compiler/onecc-docker/debian`.
But it is a temporary code so we need to add some comments about debian location.

Please add this comments to README.txt file.

> _NOTE_
> 
> The `onecc-docker` debian package should be created with `one-compiler` debian package when ONE compiler project builds. To this end, it is correct to configure the `onecc-docker` debian codes in `./infra/debian/compiler` directory. However, we are currently working on the code, so we will temporarily implement it in this location.
> 
> TODO: Merge this debian directory into `./infra/debian/compiler` code.

Reply:
How about go with `README.md` like this?
```
# onecc-docker

_onecc-docker_ is currently in incubation stage.

The onecc-docker debian package should be created with one-compiler debian package when ONE
compiler project builds. To this end, it is correct to configure the onecc-docker debian codes in
./infra/debian/compiler directory. However, we are currently working on the code, so we will
temporarily implement it in this location.

TODO: Merge this debian directory into ./infra/debian/compiler code.
```